### DISCUSSION TITLE
construction time lapse issues

### DISCUSSION THREAD

**Discussion by retroh36** [gopro/labs#272] [Q&A]
hey guys, I'm having a bit of an issue with running a construction time laps. I'm using code mPdP>07:59<20:00!585SQ~!08:00S!1R

here is the log I got 
[qrlog.txt](https://github.com/gopro/labs/files/11012829/qrlog.txt)

**Comment by retroh36** [comment]
might be worth mentioning I'm trying to do this on a GoPro hero 8 and 7

**Reply by dnewman-gpsw** [reply]
The timers in the older camera aren't very reliable, they have improved in later models.  The log file shows the error, it over slept.

[2023-03-19 08:00:07] Shutdown and Sleep:580 seconds
[2023-03-19 08:00:15] shutdown:rtc wake scheduled
[2023-03-19 08:10:31] Woke with Alarm:[08:09m01]!SQ~!08:00S!1R

Here it woke at 8:10.31am, for an alarm set for 8:09am.   So it then thought you meant the next day.  Because of these type of errors, I no longer recommend using the photo-sleep-photo style of timelapse. If you are providing external power, use this type https://gopro.github.io/labs/control/dailytl/ without the upload part.

**Reply by retroh36** [reply]
that make sense, thanks for the quick response. if the internal gopro timer is the issue, isn't it going to have this issue when it starts and stops at the beginning and end of each day? I'm trying to record construction for a house over the next 2 months running off of an 18ah battery. also should I remove the internal battery? thanks again

**Comment by Videocamguy** [comment]
Yes remove the internal battery.
